cdap-ui/app/cdap/components/Alert/index.js

```diff
@@ -12,33 +12,47 @@
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  * License for the specific language governing permissions and limitations under
  * the License.
- */
+*/
 
 import PropTypes from 'prop-types';
-
 import React, { Component } from 'react';
 import {Modal} from 'reactstrap';
+import IconSVG from 'components/IconSVG';
+
 require('./Alert.scss');
 const SUCCESS_CLOSE_TIMEOUT = 3000;
 
 export default class Alert extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showAlert: false || props.showAlert,
-      message: props.message,
-      type: props.type
-    };
-  }
+  state = {
+    showAlert: false || this.props.showAlert,
```

Comment:
nit: switch the order. `showAlert: this.props.showAlert || false`. If the first operand is true it wouldn't even go to the subsequent expression.